chain-core/src/state/tendermint.rs

```diff
@@ -40,7 +40,7 @@ impl TendermintValidatorPubKey {
 }
 
 #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Clone, Encode, Decode)]
-pub struct TendermintValidatorAddress([u8; 20]);
+pub struct TendermintValidatorAddress(pub [u8; 20]);
```

Comment:
is there a need to directly modify the internal value? If it's just getting it, them perhaps `impl Into<[u8; 20]>`?